codes/C/is_value_in_array.c

```diff
@@ -0,0 +1,17 @@
+#include <stdio.h>
+
+int value_in_array(float val, float arr[], float length) {
```

Comment:
the first and third argument here is supposed to be an `int`, not `float`. as specified in line 15 and 16.